src/components/va-alert/va-alert.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'va-alert',
+  styleUrl: 'va-alert.css',
+  shadow: true,
+})
+export class VaAlert {
+  /* Determines the icon and border/background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+  @Prop() status: string = 'info';
+
+  /**
+   * If true, renders the alert with only a background color corresponding
+   * to the status - no icon or left border.
+   */
+  @Prop() backgroundOnly: boolean = false;
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking
+   */
+  @Prop() disableAnalytics: boolean = false;
+
+  @Prop() visible: boolean = true;
+
+  /**
+   * Headline for the alert. Won't show if `backgroundOnly` is true
+   */
+  @Prop() headline: string;
+
+  /**
+   * Determines the level of the heading in the alert
+   */
+  @Prop() level: number = 3;
+
+  @Prop() closeBtnAriaLabel: string = 'Close notification';
+
+  @Prop() closeable: boolean = false;
+
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  close: EventEmitter;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private closeHandler(e: MouseEvent): void {
+    this.close.emit(e);
+  }
```

Comment:
Previously, we were able to pass a callback that got ran when the alert box was closed. Is this intended to replace that?

If so, can you give an example of how somebody might do that in React? (Context: I can't think of a way that's especially convenient, so I'm wondering if we should _additionally_ accept a callback prop and both call that function and emit the event.)

Reply:
You can still pass a callback if the component is being used in a React app, but for static HTML pages we will have to set up the event handler the old fashioned way. The [documentation](https://stenciljs.com/docs/events#using-events-in-jsx) covers this in some detail.

So, in React we would have something like:

```jsx
<va-alert headline="Blah" onClose={e => console.log('close button clicked')}>Alert content</va-alert>
```

But in a static template, we wouldn't have the `onClose` in there and instead we would have a `document.addEventListener` call to listen to the `close` event in a `<script>` tag on the page.

Reply:
Okay, that makes sense for the static site. Awesome to hear that about the `onClose` prop! :+1: 

Let's make sure we add some documentation for this so folks know they can do that. If we add JSDocs to the `@Event`, will that show up in the Storybook docs page?